src/slices/RedeemThunk.ts

```diff
@@ -48,10 +48,8 @@ export const redeemBalance = createAsyncThunk(
     } catch (e: unknown) {
       uaData.approved = false;
       const rpcError = e as IJsonRPCError;
-      if (rpcError.code === -32603 && rpcError.message.indexOf("ds-math-sub-underflow") >= 0) {
-        dispatch(
-          error("You may be trying to give more than your balance! Error code: 32603. Message: ds-math-sub-underflow"),
-        );
+      if (rpcError.message.indexOf("No redeemable balance") >= 0) {
+        dispatch(error("You have no redeemable balance"));
```

Comment:
Suggested: "You have no yield that can be redeemed."

Reply:
Will change